src/services/__tests__/MatchService.unit.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { BallChaserQueueBuilder } from "../../../.jest/Builder";
+import { createRandomMatch } from "../MatchService";
+import { mocked } from "ts-jest/utils";
+import ActiveMatchRepository from "../../repositories/ActiveMatchRepository";
+import { Team } from "../../types/common";
+import { NewActiveMatchInput } from "../../repositories/ActiveMatchRepository/types";
+
+jest.mock("../../repositories/ActiveMatchRepository");
+jest.mock("../../utils/getEnvVariable");
+
+describe("Match Service tests", () => {
+  const mockBallChasers = BallChaserQueueBuilder.many(6);
+  mockBallChasers.forEach((player) => (player.team = null));
+  const addMatchMock = mocked(ActiveMatchRepository.addActiveMatch);
+
+  it("random teams are created", async () => {
+    await createRandomMatch(mockBallChasers);
+    expect(mockBallChasers.filter((player) => player.team == Team.Orange)).toHaveLength(3);
```

Comment:
You should use `===` for strict comparison, and just for completeness can add the same check for blueTeam?

Reply:
Fixed in next commit.